akvo/rsr/models/iati_import_log.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+# Akvo RSR is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ..fields import ValidXMLTextField
+
+SEVERITY_CODE = {
+    0: _(u'information'),
+    1: _(u'critical error'),
+    2: _(u'value not saved'),
+    3: _(u'value partly saved'),
+}
```

Comment:
The severity codes should be a tuple of tuples and used for the choices attribute in IatiImportLog.severity field. [Check out the docs](https://docs.djangoproject.com/en/1.7/ref/models/fields/#django.db.models.Field.choices).

Suggested:

```python
class IatiImportLog(models.Model):
    INFORMATIONAL = 0
    CRITICAL_ERROR = 1
    VALUE_NOTSAVED = 2
    VALUE_PARTLY_SAVED = 3
    SEVERITY_CODES = (
        (INFORMATIONAL, _(u'information')),
        (CRITICAL_ERROR, _(u'critical error')),
        (VALUE_NOTSAVED, _(u'value not saved')),
        (VALUE_PARTLY_SAVED, _(u'value partly saved')),
    )
    ...
```
This way it's immediately obvious in the code that uses the severty codes what a particular severity represents, rather than having a plain integer that you have to trace back to it's origin. Also by putting these definitions inside the class, it becomes very easy to import it all since it's part of the package when you ```from iati_import_log import IatiImportLog```!